Ym2149Synth/SynthController.cpp

```diff
@@ -178,6 +178,10 @@ void SynthControllerClass::onControlChange(MidiCallbackClass * midi)
                 for(uint8_t i=0;i!=Patch[synth].patchSize;i++) {
                     midi->sendControlChange(midi->getChannel(), i+1, buffer[i]);
                 }
+                /* Send 5 dummy bytes to pad CC data */
```

Comment:
This is actually not required, my bad.
I wrongly assumed 16 bytes was required which is why you (trash80) used 16 bytes for only 11 bytes of data.

I was having a MIDI issue on Ubuntu which I can't replicate on my MacBook or on Windows.
It's an unrelated issue and this 5 byte padding can be removed.